### DISCUSSION TITLE
Freeze the compiler

### DISCUSSION THREAD

**Discussion by ghost** [IllidanS4/PawnPlus#43] [Q&A]
When i try to use it on vs code or pawn compiler its freezing i dont know if im making something wrong help please.

**Comment by IS4Code** [comment]
Try to comment out random natives from PawnPlus.inc to see if it starts working. You may start at `native ArgTag:[2]pawn_arg_pack(AnyTag:value, tag_id=tagof(value));`, this is a line that causes issues in some compilers.

**Reply by ghost** [reply]
Hello IllidanS4, 

I removed the `ArgTag:pawn_arg_pack(AnyTag:value, tag_id=tagof(value));` from my code but still same, its freezing.

**Reply by ghost** [reply]
![image](https://user-images.githubusercontent.com/73142966/111066198-ffc94d00-84bd-11eb-97ee-782e292f6e7b.png)

**Reply by ghost** [reply]
![2](https://user-images.githubusercontent.com/73142966/111066391-11f7bb00-84bf-11eb-9098-0b02ca71d5b7.png)
